### PR TITLE
fix(articleLinks): Update article links with ES links

### PR DESCRIPTION
**Pull request by RafaelDavisH**
fix(articleLinks): Update article links with ES article links

Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)

### CODE REVIEW

File: assets/config/es/footer.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const esFooter = {
       'https://www.freecodecamp.org/espanol/news/python-if-name-main/',
     article3title: 'Callback',
     article3link:
-      'https://www.freecodecamp.org/news/javascript-callback-functions-what-are-callbacks-in-js-and-how-to-use-them/',
+      'https://www.freecodecamp.org/news/what-is-a-callback-function-in-javascript/',
```
Comment (naomi-lgbt) [inline assets/config/es/footer.js]: Should this be on `/espanol/`?

Comment (RafaelDavisH) [reply assets/config/es/footer.js]: goodness! I'll get this fix

File: assets/config/es/footer.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ const esFooter = {
       'https://www.freecodecamp.org/espanol/news/product-design-explicado-en-un-espanol-sencillo/',
     article14title: 'Digital Design',
```
Comment (naomi-lgbt) [inline assets/config/es/footer.js]:
```suggestion
    article14title: 'Diseño Digital',
```

One more - should this title be Spanish?

Comment (RafaelDavisH) [reply assets/config/es/footer.js]: 🤦🏽‍♂️ Thanks Nick

**Review by scissorsneedfoodtoo — APPROVED** [review]
Everything LGTM :+1: :+1: 

Thanks for these updates @RafaelDavisH. Also, thanks for your thorough review @nhcarrigan.

**Pull request merged**